jac-uk/digital-platform
# Update assessments functions

## What's included?
1. Add **assessor type** when create a new assessment.
2. Add `resetAssessment` to change assessment status.
3. Update parameters passed into `cancelAssessment` and `testAssessmentNotification`.
4. Update Firestore index to support assessor type filter.
5. Update Firestore rules to allow update assessment status.
6. Send submit notification when assessments are completed.

Note: this PR is related to [#1690 Update independent assessments UI](https://github.com/jac-uk/admin/pull/1696).

## Who should test?
✅ Developers

## How to test?
1. Run functions locally.
2. On admin site, check out branch `feature/1690-update-independent-assessments-ui`.
3. Follow the test steps in this PR [#1690 Update independent assessments UI](https://github.com/jac-uk/admin/pull/1696).

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context


## Related permissions
- No permission changes required
